### PR TITLE
feat: remove property mapping for other standard events in fb pixel

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## Description of the change

> We are removing the property mapping that was mandatory in order to send the mapped properties here inside Other Standard events of out FB Pixel integration.

Removing this, will bring standard events in line with custom events, in terms of properties being sent along with the event.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: test/__tests__/data/facebook_pixel_output.json [hunk 1]
```diff
@@ -199,7 +199,7 @@
...
+          "{\"user_data\":{\"external_id\":\"5994471abb01112afcc18159f6cc74b4f511b99806da59b3caf5a9c173cacfc5\",\"client_user_agent\":\"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36\"},\"event_name\":\"ViewContent\",\"event_time\":1567333011,\"event_id\":\"ec5481b6-a926-4d2e-b293-0b3a77c4d3be\",\"action_source\":\"website\",\"custom_data\":{\"category\":\"cat 1\",\"list_id\":\"1234\",\"filters[0].type\":\"department\",\"filters[0].value\":\"beauty\",\"filters[1].type\":\"price\",\"filters[1].value\":\"under\",\"sorts[0].type\":\"price\",\"sorts[0].value\":\"desc\",\"testDimension\":true,\"testMetric\":true,\"content_ids\":[\"cat 1\"],\"content_type\":\"product_group\",\"contents\":[{\"id\":\"cat 1\",\"quantity\":1}],\"content_category\":\"cat 1\",\"value\":0,\"currency\":\"USD\"}}"
```
Comment (Gauravudia) [inline test/__tests__/data/facebook_pixel_output.json:202]: Does Facebook support this type of key? 
`filters[0].type`

Comment (yashasvibajpai) [reply test/__tests__/data/facebook_pixel_output.json:202]: Yes.

**Review by Gauravudia — APPROVED** [review]

File: src/v0/destinations/facebook_pixel/utils.js [hunk 2]
```diff
@@ -192,8 +183,8 @@ const transformedPayloadData = (
         delete clonedCustomData[eventProp];
       }
     }
-    const isCustomProperty = customEventProperties[eventProp] || false;
-    if (isStandard && !isCustomProperty && !isDefaultPiiProperty) {
+
+    if (isStandard && isDefaultPiiProperty) {
```
[the remark this answers is not in the document]
Comment (yashasvibajpai) [reply src/v0/destinations/facebook_pixel/utils.js:187]: fixed

**Review by ItsSudip — APPROVED** [review]

**Comment by yashasvibajpai** [comment]
Closing in lieu of https://github.com/rudderlabs/rudder-transformer/pull/2215

**Pull request closed without merging**